homeassistant/components/comelit/coordinator.py

```diff
@@ -68,13 +68,16 @@ def platform_device_info(self, device: ComelitSerialBridgeObject) -> dr.DeviceIn
     async def _async_update_data(self) -> dict[str, Any]:
         """Update device data."""
         _LOGGER.debug("Polling Comelit Serial Bridge host: %s", self._host)
+
+        devices = {}
         try:
             await self.api.login()
+            devices = await self.api.get_all_devices()
```

Comment:
Can you not just return here?
```suggestion
            return await self.api.get_all_devices()
```

Reply:
Done